src/components/BottomMenu.tsx

```diff
@@ -82,7 +83,10 @@ export default function BottomMenu({
           <MenuItem
             label={formatMessage(noticeAndDoNotSellMessages.doNotSellLabel)}
             type="button"
-            onClick={() => handleSetViewState(ViewState.CompleteOptions)}
+            onClick={(event) => {
+              airgap.setConsent(event, { SaleOfInfo: false });
+              handleSetViewState(ViewState.CompleteOptions);
```

Comment:
Can we split this functionality into a separate PR? This needs a bit of design review, as navigating to `CompleteOptions` doesn't make much sense anymore. Given the opt out is being set here, it should be confirmed in the following view. This view is a slightly confusing place to land, as there's a save button.

Reply:
Ok, fine to split this out 👍

Reply:
Thank you! Mind also opening a ticket in Height for the design review?

Reply:
Done: https://transcend.height.app/T-14275

Reply:
Thanks!